src/components/Tag.js

```diff
@@ -75,11 +85,11 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
             '?search[stringTraits][0][name]=' +
             title +
             '&search[stringTraits][0][values][0]=' +
-            text
+            value ?? text
```

Comment:
I had to use a value (originalValue) that can be used for searching on OpenSea. I added a fallback where only text was used in `Tag` usages so that others are left as they were.

Reply:
Should I document it with a comment or rename value to originalValue for example?

Reply:
Renaming to originalValue sounds better to me.

Reply:
are you sure it works then?

Reply:
@osdnk  So in terms of URL conversion.
`https://google.com` -> `google.com` every string value that starts with https:// gets the https:// prefix cut everything else stays.